sklearn/tree/_classes.py

```diff
@@ -783,6 +783,11 @@ class DecisionTreeClassifier(ClassifierMixin, BaseDecisionTree):
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
+    The :meth:`predict()` function operates using the :func:`numpy.argmax()`
+    function on the outputs of :meth:`predict_proba()`. This means that in case
+    of a tied, highest predicted probability, the classifier will predict the
+    tied class with the lowest index.
```

Comment:
A few more changes:

* `predict` is a method, not a function.
* The parentheses are not required. In fact, they are removed from the documentation for simplicity.  

```suggestion
    The :meth:`predict` method operates using the :func:`numpy.argmax`
    function on the outputs of :meth:`predict_proba`. This means that in
    case of a tied, highest predicted probability, the classifier will predict
    the tied class with the lowest index.
```